app/views/devise/sessions/new.html.erb

```diff
@@ -13,5 +13,6 @@
   <% end %>
 
   <%= render "devise/shared/links" %>
+  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
```

Comment:
Implementation note: Add query string param here if `resource.errors[:email]` is present (or whatever is the right syntax here)